### ISSUE TITLE
Issue installation in Laravel 5.4

### ISSUE THREAD

**Issue by kossa** [ignited/laravel-omnipay#36]
Hi,

when I installed the package in laravel 5.4 I had :
```
[Symfony\Component\Debug\Exception\FatalThrowableError]              
Call to undefined method Illuminate\Foundation\Application::share()  
```

I think problem with share method, please check : https://laravel.com/docs/5.4/upgrade
> #### share Method Removed
>The share method has been removed from the container. This was a legacy method that has not been documented in several years. If you are using this method, you should begin using the  singleton method instead:

**Comment by alexw23** [comment]
Fixed in #37 please download `2.3.*`

**Comment by kossa** [comment]
Thank you @alexw23 (y)
